src/flavors/__tests__/__snapshots__/openapi.test.ts.snap

```diff
@@ -356,6 +354,7 @@ Array [
     "constraints": Object {
       "isArray": "requiredPrimitiveArrayValue must be an array",
       "isDefined": "requiredPrimitiveArrayValue should not be null or undefined",
+      "isNumber": "each value in requiredPrimitiveArrayValue must be a number conforming to the specified constraints",
```

Comment:
This change is due to the previous implementation using `@IsArray` and not giving `class-validator` enough info.